vision/module/location.py

```diff
@@ -377,6 +385,23 @@ def showCircles(self) -> None:
         cv2.waitKey(0)
         cv2.destroyAllWindows()
 
+    def saveCircleImage(self, filename, folder) -> None:
```

Comment:
See above comment, can remove the folder parameter. Also, make sure to "type" your parameters. Example:

`def saveCircleImage(self, filename: str) -> None:`